package.json

```diff
@@ -21,7 +21,7 @@
     "bootstrap-switch": "3.3.2",
     "colors": "1.1.2",
     "css-loader": "^0.26.1",
-    "eonasdan-bootstrap-datetimepicker": "~4.15",
+    "eonasdan-bootstrap-datetimepicker": "~4.17",
```

Comment:
See https://github.com/girder/girder/pull/1893/commits/0cf0f24b9c85acc7019e3f3b3b50712e99a272b8

Fix would be to change the expected result to match the input, which I think is reasonable things to do. See: https://github.com/girder/girder/blob/834cf29acc3538b38ba24e6e7203852924110843/clients/web/test/spec/dateTimeWidgetSpec.js#L150-L153  which